pkg/distro/defs/loader_test.go

```diff
@@ -533,10 +533,14 @@ image_types:
       hostname: "foo"
       locale: "C.UTF-8"
       timezone: "DefaultTZ"
+      default_kernel_name: kernel
       condition:
         version_less_than:
           "2":
             timezone: "OverrideTZ"
+          # test-distro is version "1" (no minor) so considered "1" is > "1.4"
```

Comment:
I had to read this twice before understanding what it meant, but it is indeed correct.